parquet/metadata/file.go

```diff
@@ -275,6 +278,9 @@ func NewFileMetaData(data []byte, fileDecryptor encryption.FileDecryptor) (*File
 // Size is the length of the raw serialized metadata bytes in the footer
 func (f *FileMetaData) Size() int { return f.metadataLen }
 
+// SourceSz is the total size of the source file
+func (f *FileMetaData) SourceSz() int64 { return f.footerOffset }
```

Comment:
Because the FileMetadata can potentially be separate from the file itself, it might make more sense to name this differently or at least use a different description for the comment.